test/functional/mempool_package_rbf.py

```diff
@@ -0,0 +1,595 @@
+        self.assert_mempool_contents(expected=package_txns)
+
+        self.generate(node, 1)
+
+    def test_package_rbf_additional_fees(self):
+        self.log.info("Check Package RBF must increase the absolute fee")
+        node = self.nodes[0]
+        coin = self.coins.pop()
+
+        # avoid parent pays for child anti-DoS checks
+        fee_delta = DEFAULT_FEE / 2
+
+        package_hex1, package_txns1 = self.create_simple_package(coin, parent_fee=DEFAULT_FEE - fee_delta, child_fee=DEFAULT_FEE + fee_delta, heavy_child=True)
+        assert_greater_than_or_equal(1000, package_txns1[-1].get_vsize())
+        node.submitpackage(package_hex1)
+        self.assert_mempool_contents(expected=package_txns1)
+
+        # Package 2 has a higher feerate but lower absolute fee
+        package_hex2, package_txns2 = self.create_simple_package(coin, parent_fee=DEFAULT_FEE - fee_delta, child_fee=DEFAULT_FEE + fee_delta - Decimal("0.00000001"))
+        pkg_results2 = node.submitpackage(package_hex2)
+        assert "package RBF failed: insufficient anti-DoS fees, rejecting replacement" in pkg_results2["package_msg"]
+        self.assert_mempool_contents(expected=package_txns1)
+
+        # Package 3 has a higher feerate and absolute fee
+        package_hex3, package_txns3 = self.create_simple_package(coin, parent_fee=DEFAULT_FEE, child_fee=DEFAULT_FEE * 3)
+        node.submitpackage(package_hex3)
+        self.assert_mempool_contents(expected=package_txns3)
+        self.generate(node, 1)
+
+        self.log.info("Check Package RBF must pay for the entire package's bandwidth")
+        coin = self.coins.pop()
+        package_hex4, package_txns4 = self.create_simple_package(coin, parent_fee=DEFAULT_FEE, child_fee=DEFAULT_FEE)
+        node.submitpackage(package_hex4)
+        self.assert_mempool_contents(expected=package_txns4)
+        package_hex5, package_txns5 = self.create_simple_package(coin, parent_fee=DEFAULT_FEE - fee_delta, child_fee=DEFAULT_FEE + fee_delta + Decimal("0.00000001"))
+        pkg_results5 = node.submitpackage(package_hex5)
+        assert_equal(f"package RBF failed: insufficient anti-DoS fees, rejecting replacement {package_txns5[1].rehash()}, not enough additional fees to relay; 0.00000001 < 0.00000{sum([tx.get_vsize() for tx in package_txns5])}", pkg_results5["package_msg"])
+
+        self.assert_mempool_contents(expected=package_txns4)
+        self.generate(node, 1)
+
+        self.log.info("Check Package RBF must have strict cpfp structure")
+        coin = self.coins.pop()
+        package_hex5, package_txns5 = self.create_simple_package(coin, parent_fee=DEFAULT_FEE, child_fee=DEFAULT_FEE)
+        node.submitpackage(package_hex5)
+        self.assert_mempool_contents(expected=package_txns5)
+        package_hex6, package_txns6 = self.create_simple_package(coin, parent_fee=DEFAULT_FEE + fee_delta, child_fee=DEFAULT_FEE - (fee_delta / 2))
```

Comment:
Given that `fee_delta` is a multiple of `DEFAULT_FEE` and you're using `fee_delta / 2` here... I think the general readability of this test would be improved if you created a constant base unit `FEE` which was equal to, say, `DEFAULT_FEE / 10` (or `104 * 10` which would give most of the transactions whole number feerates), and then used `FEE`, `2*FEE`, `8*FEE`, etc. in all the tests.

Reply:
I created `DEFAULT_CHILD_FEE` and used that pretty much everywhere it made sense